browser/model/virtualbox.js

```
@@ -118,9 +118,12 @@ class VirtualBoxInstallWindows extends VirtualBoxInstall {
 
   installAfterRequirements(progress, success, failure) {
     let installer = new Installer(this.keyName, progress, success, failure);
-    return installer.exec(
-      [`"${this.downloadedFile}"`, '--extract', '-path', `"${this.installerDataSvc.virtualBoxDir()}"`, '--silent'].join(' ')
-    ).then(() => {
+    return this.importCertificate(installer
+    ).catch(() => {
+      // continue even if certificate fails to import automatically
```

I don't like empty catch blocks, lets at least log warning here.